boards/phytec/mimx93_phyboard_nash/board.yml

```
@@ -0,0 +1,5 @@
+board:
+  name: mimx93_phyboard_nash
```

I would suggest to remove `mimx93` from the board name. SoC vendors normally call their EVM, EVK, ... after the silicon itself to better promote it. We have unique names for our boards (`phyBOARD-Nash`).

I also removed them from our TI boards since hardware model v2 allows multiple SoC in one board directory. It just looks cleaner IMO:

```
phyboard_nash/mimx9352/a55
phyboard_nash/mimx9352/m33
```


```suggestion
  name: phyboard_nash
```

---

I followed the pattern that we used for the imx8 SoCs but I agree that it looks cleaner to omit the prefix, now that there is HWM2.

Will can a PR in the next days that aligns the naming for the PHYTEC imx8 platforms.

---

Thanks!

---

As reference I'll mention the PR here: https://github.com/zephyrproject-rtos/zephyr/pull/79575